src/styles/views/Lobby.scss

```diff
@@ -0,0 +1,106 @@
+@import "../theme";
+
+/* The '&' is a shorthand for the containing element,
+  so for example a '&.user-list' inside .game will compile to
+  .game.user-list
+
+  for more information visit https://sass-lang.com/guide
+ */
+
+.lobbypage {
+
+  &.container {
+    float: left;
+    background: $background;
+    padding: 1.5em;
+    border-radius: $borderRadius;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    box-shadow: $dropShadow;
+    margin-left: 30px;
+  }
+
+  & p,
+  h2 {
+    margin: 0.5em 0;
+  }
+}
+
+.gamemode {
+
+  &:hover {transform: translateY(-2px);}
+  &.container {
+    text-align: start;
+    margin: 0.5em 0;
+    width: auto;
+    padding: 20px;
+    border-radius: $borderRadius;
+    display: flex;
+    align-items: center;
```

Comment:
Add `flex-direction: column;`
This will make the desc go below the game mode name.